aspnetcore/signalr/hubcontext.md

```diff
@@ -0,0 +1,60 @@
+---
+title: SignalR HubContext
+author: rachelappel
+description: Overview of the use of SignalR's HubContext
+manager: wpickett
+monikerRange: '>= aspnetcore-2.1'
+ms.author: rachelap
+ms.custom: mvc
+ms.date: 6/12/18
+ms.prod: aspnet-core
+ms.technology: aspnet
+ms.topic: article
+uid: signalr/HubContext
+---
+
+# Sending Messages From Outside a Hub
+
+By [Mikael Mengistu](https://github.com/mikaelm12)
+
+The SignalR Hub is the core abstraction for sending messages to clients connected to our SignalR server. However, it is also possible to send messages from other places in your application using the IHubContext service. Here we'll look at how to access a SignalR IHubContext to send notification to clients from outside of a hub.
+
+[View or download sample code](https://github.com/aspnet/Docs/tree/master/aspnetcore/signalr/hubcontext/sample/) [(how to download)](xref:tutorials/index#how-to-download-a-sample)
+
+## Get an instance of `HubContext`
```

Comment:
The User-facing type is `IHubContext<T>`, let's use that term so they can match it up with what they see in their code. That applies throughout the doc (and it should either always be in `code ticks` or never in `code ticks` :))